js/pages/questionnaire.js

```diff
@@ -14,200 +13,194 @@ let modules;
 const questDiv = "questionnaireRoot";
 
 const importQuest = async () => {
+    const url = questConfig[location.host];
+    
+    try {    
+        const { transform } = await import(url);
 
-    let url = questConfig[location.host];
+        if (!transform) {
+            throw new Error('Transform error while importing quest');
```

Comment:
incredibly nit, but might suggest the follow verbiage:

`Error loading transform module from Quest`

Reply:
updated